### PR TITLE
Helm charts and helmfiles

### PR DESCRIPTION
**Pull request by dylanbannon**
Helm charts and helmfiles for all components of the project to date.

### CODE REVIEW

File: conf/charts/frontend-redis-interface/values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# service.yaml
+service_name: "fe-rd-service"
+service_port: 8080
+service_type: "ClusterIP"
+
+# secrets.yaml and hemlfile.d/frontend.yaml
+env:
+  REDIS_HOST: "redis-master"
+  REDIS_PORT: "6379"
+  secret:
```
Comment (osterman) [inline conf/charts/frontend-redis-interface/values.yaml]: Remove leading whitespace?

File: conf/charts/frontend-redis-interface/values.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    AWS_ACCESS_KEY_ID: "change_me"
+    AWS_SECRET_ACCESS_KEY: "change_me"
+    AWS_S3_BUCKET: "change_me"
+secrets:
```
Comment (osterman) [inline conf/charts/frontend-redis-interface/values.yaml]: Deprecate?

**Comment by osterman** [comment]
Fix merge conflict

File: conf/charts/frontend-redis-interface/values.yaml (hunk 1 above)
Comment (dylanbannon) [reply conf/charts/frontend-redis-interface/values.yaml]: I'm a little confused. Most of these lines don't have leading whitespace. Are you talking about the variables under the `env` section.

File: conf/charts/frontend-redis-interface/values.yaml (hunk 2 above)
Comment (dylanbannon) [reply conf/charts/frontend-redis-interface/values.yaml]: Sorry, I'm unclear here, too.

File: conf/charts/frontend-redis-interface/values.yaml (hunk 1 above)
Comment (osterman) [reply conf/charts/frontend-redis-interface/values.yaml]: We will address this in a separate PR.

File: conf/charts/frontend-redis-interface/values.yaml (hunk 2 above)
Comment (osterman) [reply conf/charts/frontend-redis-interface/values.yaml]: We will address this in a separate PR.

File: conf/charts/frontend/values.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+app_name: "frontend"
+
+#deployment.yaml
+replicas: 1
+image_name: "vanvalenlab/react_interface"
+image_version: "latest"
```
Comment (osterman) [inline conf/charts/frontend/values.yaml]: In a new PR, pin this to a release

File: conf/charts/frontend/values.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  EXPRESS_HOST: "fe-rd-service"
+  EXPRESS_PORT: "8080"
+  MODEL_NAME: "mibi"
+  MODEL_VERSION: "0"
+  AWS_REGION: "us-east-1"
+  secret:
+    AWS_ACCESS_KEY_ID: "change_me"
+    AWS_SECRET_ACCESS_KEY: "change_me"
+    AWS_S3_BUCKET: "change_me"
+secrets:
```
Comment (osterman) [inline conf/charts/frontend/values.yaml]: Deprecate

File: conf/charts/redis-poller/values.yaml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+# deployment.yaml
+deployment_name: "redis-poller-deployment"
+replicas: 1
+image_name: "vanvalenlab/redis-poller"
+image_version: "latest"
+container_name: "redis-poller-container"
+env:
+  DEBUG: "true"
+  INTERVAL: "specify_me"
+  REDIS_HOST: "overwrite_this"
```
Comment (osterman) [inline conf/charts/redis-poller/values.yaml]: Provide a sane default

File: conf/charts/redis-poller/values.yaml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+# deployment.yaml
+deployment_name: "redis-poller-deployment"
+replicas: 1
+image_name: "vanvalenlab/redis-poller"
+image_version: "latest"
+container_name: "redis-poller-container"
+env:
+  DEBUG: "true"
+  INTERVAL: "specify_me"
```
Comment (osterman) [inline conf/charts/redis-poller/values.yaml]: provide a sane default (integer)

File: conf/charts/tf-serving-redis-interface/values.yaml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+service_port: 8080
+service_type: "ClusterIP"
+
+# secrets.yaml and hemlfile.d/frontend.yaml
+env:
+  secret:
+    AWS_ACCESS_KEY_ID: "change_me"
+    AWS_SECRET_ACCESS_KEY: "change_me"
+    AWS_S3_BUCKET: "change_me"
+secrets:
```
Comment (osterman) [inline conf/charts/tf-serving-redis-interface/values.yaml]: Deprecate

**Pull request merged**